tests/test_dnssec.py

```diff
@@ -926,19 +926,12 @@ def testInvalidDigestLengthCDS0(self):  # type: () -> None
                 self.assertEqual(msg, str(cm.exception))
 
     def testKeyToDNSKEY(self):  # type: () -> None
-        rsa2048_key = load_pem_private_key(PRIVATE_KEY_RSA_2048.encode(), password=None)
-        rsa4096_key = load_pem_private_key(PRIVATE_KEY_RSA_4096.encode(), password=None)
-        ecdsa_p256_key = load_pem_private_key(PRIVATE_KEY_EC_P256.encode(), password=None)
-        ecdsa_p384_key = load_pem_private_key(PRIVATE_KEY_EC_P384.encode(), password=None)
-        ed25519_key = load_pem_private_key(PRIVATE_KEY_ED25519.encode(), password=None)
-        ed448_key = load_pem_private_key(PRIVATE_KEY_ED448.encode(), password=None)
-    
-        rsa2048_rdata = dns.dnssec.key_to_dnskey(rsa2048_key.public_key(), dns.dnssec.Algorithm.RSASHA256)
-        rsa4096_rdata = dns.dnssec.key_to_dnskey(rsa4096_key.public_key(), dns.dnssec.Algorithm.RSASHA256)
-        ecdsa_p256_rdata = dns.dnssec.key_to_dnskey(ecdsa_p256_key.public_key(), dns.dnssec.Algorithm.ECDSAP256SHA256)
-        ecdsa_p384_rdata = dns.dnssec.key_to_dnskey(ecdsa_p384_key.public_key(), dns.dnssec.Algorithm.ECDSAP384SHA384)
-        ed25519_rdata = dns.dnssec.key_to_dnskey(ed25519_key.public_key(), dns.dnssec.Algorithm.ED25519)
-        ed448_rdata = dns.dnssec.key_to_dnskey(ed448_key.public_key(), dns.dnssec.Algorithm.ED448)
+        for tk in test_dnskeys:
+            print(tk.command)
```

Comment:
debug output

Reply:
Pytest will discard all output to
STDOUT unless the test fails, so it might be useful to keep.